go/pkg/apis/enricher/framework/java/wildfly_detector.go

```diff
@@ -0,0 +1,72 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc.
+ ******************************************************************************/
+package enricher
+
+import (
+	"context"
+	"encoding/xml"
+	"strings"
+
+	"github.com/redhat-developer/alizer/go/pkg/apis/model"
+	utils "github.com/redhat-developer/alizer/go/pkg/utils"
+)
+
+type WildFlyDetector struct{}
+
+type Wildfly_Standalone_Xml struct {
+	SocketBindingGroup struct {
```

Comment:
Same question here. No standalone.xml

Reply:
I'm resolving this one. Let's follow up on the above thread :)